torch/csrc/jit/passes/quantization.cpp

```diff
@@ -986,10 +986,20 @@ void InsertObserversHelper::preprocess(
   }
 }
 
-// Returns true if the value is the weight to LSTM operator.
-bool isDynamicLSTMWeight(Value* v, Use use, bool is_dynamic) {
-  return is_dynamic && use.user->kind() == Symbol::aten("lstm") &&
-      (use.offset == 2);
+bool valueQuantizable(Use use, bool is_dynamic) {
+  Node* n = use.user;
+  // static quantization
+  if (n->kind() == prim::CallFunction &&
+      getFuncName(n->inputs()[0]) == "batch_norm") {
+    return use.offset == 1;
+  }
```

Comment:
could you refactor this further to take a list of tuple of (name, offset) and check if the use is matching the name and offset?

Reply:
some similar functions we already implemented: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L707, https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization.cpp#L713, please feel free to refactor these as well as you see fit

Reply:
I've added the list of tuple in the function itself. I didn't want to spend too much time refactoring the other code, so I used it in the best way I saw fit, given that the conditions we were returning on in both cases was different.